### ISSUE TITLE
making CI tests faster

### ISSUE THREAD

**Issue by sibirrer** [LSST-strong-lensing/slsim#91]
The CI tests are taking several minutes and using up a lot of computational time/cost which makes the development less convenient. I was wondering whether we can make tests such that they do not use so long. It's usually few tests that end up using most of the time. Perhaps it's the skypy catalogue generation @nkhadka21

**Comment by nkhadka21** [comment]
Yes, this is true. I think we should try to generate some skypy data and put them in the TestData so that we can use them again and again.
